### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by dannyweldon** [LINBIT/csync2#17]
Any chance of a new release to pick up the latest fixes for version 2?  Especially to include 999f153ae042910e2d8a79f749be66d4ca11296c which is the main problem we see from time to time.

**Comment by lge** [comment]
The mentioned commit is over 2.5 years old, if your favorite distribution did not pick that up meanwhile, that is not really csync2's fault ;-)
But okay, I'll add a release tag for 2.1 within the next days, pretending that would carry some meaning. Because, why not. Thank you for the friendly reminder!

**Comment by dannyweldon** [comment]
Thank you!

**Comment by jirutka** [comment]
> if your favorite distribution did not pick that up meanwhile, that is not really csync2's fault ;-)

No, it is. All distributions normally package **released** software **versions**, not an arbitrary commit in the development branch. There are many good reasons for this. The last csync2 release is already 11 years old.

**Comment by ROBERT-MCDOWELL** [comment]
and works very well...

**Comment by jirutka** [comment]
Actually, it doesn’t. We had to backport several patches from master to make it work.

**Comment by ROBERT-MCDOWELL** [comment]
maybe fork if your PR does not pass?
